MCprep_addon/MCprep_resources/Languages/mcprep.pot

```diff
@@ -0,0 +1,523 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2024-01-01 10:18-0600\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: MCprep_addon/mcprep_ui.py:98
```

Comment:
I will admit, I'm both a bit concerned and confused by this file.

The concern: if we add one new line at the start of each file, does that already invalidate every single line in every single po file, since the lineno is no longer accurate?

The confusion: Are these line numbers somehow automatically generated, or did you have to hand author these line numbers? If hand authored, then I have no idea how we'll keep this up to date. Seems like we should be able to get away with the .po key translation on its own for any given string, shouldn't we? I recall the built in way of blender handling translations was by doing something like auto recognizing a given phrase is used for translation, and then automatically swapping it for us so long as that exact phrase was present in the translations dictionary.

I'm all for using more standardized cross-project ways of managing translations, this feels like it's going to be a huge amount of work and much more cumbersome than what we had in prior iterations with just key-value pair lookups.

Reply:
The line numbers are autogenerated by `xgettext`. They don't have actual meaning, beyond just giving an idea of where in the source code the text comes from. The actual import parts are lines like these:
```po
msgid "Restart blender"
msgstr ""
```

Which are identical to our old system of key-value pairs, just in a more standard format, with parsing handled by Python's `gettext` module. There's no really "built-in" way of doing translations in Blender, but Blender itself does use PO files as well, so I think we're on the right track.